packages/trader/src/App/Components/Elements/ContractAudit/__tests__/contract-details.spec.js

```diff
@@ -55,7 +55,7 @@ describe('ContractDetails', () => {
                 duration={1}
                 exit_spot={123}
                 is_vanilla={true}
-                number_of_contracts={10}
+                display_number_of_contracts={'10'}
```

Comment:
we can just do this no? `'10'`

Reply:
Just asking: `<ContractDetails />` doesn't receive `currency` and `display_number_of_contracts` from props directly, they are extracted from contract_info if I understand it correctly. Then why do we need to pass them as props in test cases?

Reply:
@kate-deriv yeah, I think this is not required along with currency, I'll just remove it from affected lines